Automattic/wp-calypso
# Jetpack Checklist: De-dupe product feature cards

#### Changes proposed in this Pull Request

Clean up the security related cards below the checklist, and add any actionable links to the checklist items themselves after the feature has been turned on.

**Update** Per @sirreal's [suggestion](https://github.com/Automattic/wp-calypso/pull/32649#issuecomment-488641987), cards are no longer removed as part of this PR but broken out into two new ones, #32768 and #32773.

Implementing the following features from #32557:

> 1. Brute force attack feature:
>    1. [x] remove card below checklist.
>    2. [x] add "configure" link to the completed checklist item that sends customers to the appropriate settings within calypso.
> 2. backup and scan feature:
>    1. [x] remove site security card below checklist.
>    2. [ ] add link to "view security dashboard" in the completed checklist item.
>       - [x] for now it should take the customer to their respective vaultpress dashboard.
>       - [ ] in the future, we'll send them to the calypso UI formerly known as Rewind (cc @rcoll ) :arrow_right: #32698 
> 3. spam filtering feature:
>    1. [x] remove spam filtering card below checklist
>    2. [ ] add "view spam stats" to its respective completed checklist item
>       - [x] to take customers to their akismet dashboard within wp-admin. I think this will show a little more value for customers.
>       - [ ] Note: have this link open in new window since there’s currently no way for customers to return to wordpress.com from akismet dash :arrow_right: #32697
> 4. concierge orientation card
>    1. [x] @chaselivingston had mentioned that concierge may be shutting down for a bit in Q2. If that's the case, and if this block won't auto disappear, we should hide it for now. Also in the future, I think concierge orientation should be part of the checklist.
> 5. update completed tasks action link verbiage
>    1. [x] currently completed tasks say "change" to the right of the checklist. Paired with the language to the checklist item, the word "change" make it seems like a customer can only turn something on or off. It doesn't communicate that these features have settings that can be configured.

And, from the [illustration](https://user-images.githubusercontent.com/214813/56674490-27278e80-6688-11e9-9b36-5f377316150e.jpg):

> - [ ] Brute force attack feature: :arrow_right: #32699
>  - Add Guided Tour
>  - Auto-expand

Unticked checkboxes are features that are more involved to implement. I've filed issues to tackle them separately.

#### Testing instructions

- Run this Calypso branch locally
- Create a new [`jurassic.ninja`](https://jurassic.ninja/) site
- Connect it to WP.com: Use the Connect button's link target and append `&calypso_env=development`
- Pick a paid plan
- Verify that the Thank You page has been changed as specified by the list above.

Fixes #32557
